levels/ending/geo.c

```diff
@@ -21,9 +21,7 @@ const GeoLayout ending_geo_area_1[] = {
       GEO_OPEN_NODE(),
          GEO_NODE_ORTHO(100),
          GEO_OPEN_NODE(),
-#ifdef VERSION_EU
-            GEO_BACKGROUND_COLOR(0x0001),
-#endif
+            GEO_BACKGROUND_COLOR(GPACK_RGBA5551(0, 0, 0, 1)),
```

Comment:
Why wasn't this in Shindou? 🤔

Reply:
This was probably originally found in EU when they were making the end cake screen change with the selected language, and since it didn't change in shindou they probably just reverted all of the leveldata. Just a guess though.